### PR TITLE
INS-2487: Added error handler

### PR DESCRIPTION
**Pull request by pol9kov**
**- What I did**
Added error handler 

**- How I did it**
Added error handler 

**- How to verify it**
Return error to RegisterNode method in NodeDomain contract

**- Description for the changelog**

### CODE REVIEW

File: cmd/insolar/certgen.go [hunk 1]
```diff
@@ -59,6 +60,10 @@ func extractReference(response []byte, requestTypeMsg string) insolar.Reference
 	r := RegisterResult{}
 	err := json.Unmarshal(response, &r)
 	checkError(fmt.Sprintf("Failed to parse response from '%s' node request", requestTypeMsg), err)
+	if r.Error != "" {
```
Comment (ivanshibitov) [inline cmd/insolar/certgen.go]: Use checkError here

Comment (pol9kov) [reply cmd/insolar/certgen.go]: checkError method signature doesn't fit

File: cmd/insolar/certgen.go [hunk 2]
```diff
@@ -59,6 +60,10 @@ func extractReference(response []byte, requestTypeMsg string) insolar.Reference
...
+		fmt.Printf("Error while '%s' occured : %s \n", requestTypeMsg, r.Error)
+		os.Exit(1)
+	}
 	if verbose {
```
Comment (ivanshibitov) [inline cmd/insolar/certgen.go]: I think it should be before your check

File: cmd/insolar/certgen.go (hunk 1 above)
Comment (ivanshibitov) [reply cmd/insolar/certgen.go]: ok

**Pull request merged**